package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ninshiki_yolo</name>
+  <version>0.0.0</version>
```

Comment:
I actually prefer to start project's version from `0.1.0`, but it's actually fine to use that.